typescript/scripts/build.sh

```diff
@@ -13,5 +13,13 @@ lang_root_dirpath="$(dirname "${script_dirpath}")"
 # # ==================================================================================================
 # #                                             Main Logic
 # # ==================================================================================================
-find_results="$(find ${lang_root_dirpath}/lib -name "*.ts")"
-tsc ${find_results} --outFile ${lang_root_dirpath}/build/output.js --module system --moduleResolution node --target es2015
+#find_results="$(find ${lang_root_dirpath} -name "*.ts" -not -path "${lang_root_dirpath}/kurtosis_core_rpc_api_bindings/*" -not -path "${lang_root_dirpath}/node_modules/*")"
+
+tsc ${lang_root_dirpath}/index.ts --outDir ${lang_root_dirpath}/build/ --downlevelIteration true --declaration true
```

Comment:
Minor preference for `tsconfig.json`, but don't worry about it for now - this is fine